### DISCUSSION TITLE
Request: 通知されたページへアクセスを通知一覧ページと同様に遷移したい

### DISCUSSION THREAD

**Discussion by eyelly** [growilabs/growi#7732] [Ideas / Requests]
## Informations
**Is your feature request related to a problem? Please describe.**
以下画像の状態だと、2回クリックしないとページに遷移しない。
![image](https://user-images.githubusercontent.com/14306698/207217129-d6d00e13-4dda-4152-bdcf-b6c9c8d6d730.png)

以下画像の通知一覧ページであれば未読状態でも1回クリックで遷移する。
![image](https://user-images.githubusercontent.com/14306698/207218761-4513e5d3-6a78-46fc-97e6-160324371a99.png)

**Describe the solution you'd like**
A clear and concise description of what you want to realization.
1回クリックで通知の既読とページ遷移をしたい。

**Describe alternatives you've considered**
通知一覧ページと同様の動きの方が利便性が高い。
未読/既読を明確に分けたいのであれば、既読/未読を切り替えるボタン、ページ遷移するボタンを分けて表示しても良いかもしれない。

ご指摘等あればコメントお願いします。

**Comment by yuki-takei** [comment]
@eyelly Thank you for your report. I think it might be a bug. Would you add the versions of your system?
ご連絡ありがとうございます。バグと言ってもいい挙動かもしれません。可能であればバグ報告用のテンプレートにあるように、システムバージョン等を追記いただけると助かります。

Last month, v6 has released. So we will investigate whether the issue is reproduced in v6 and consider about v5 also.
また、先月 v6 がリリースされたのでこの問題が最新版ではどうなっているのかを確認した後、v5 向けの修正を行うかを検討したいと思います。
